src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -70,7 +70,7 @@ export const WorldsDashboard: React.FC<WorldsDashboardProps> = ({ userId }) => {
                   <SectionTitle>My worlds</SectionTitle>
                   <HeaderButton
                     to={ADMIN_IA_WORLD_CREATE_URL}
-                    extraClasses="text-white bg-sparkle rounded-full"
+                    extraClasses="text-white bg-sparkle"
```

Comment:
Please don't use any injected classes. Put the differences behind a variant prop or a new component. It allows for more uniform UI/UX

```suggestion
```

Reply:
added variants: primary, secondary, multicolor: https://github.com/sparkletown/sparkle/pull/2812/commits/848dd4d9d93b293313e1e193e695d17e508fd22f

<img width="768" alt="image" src="https://user-images.githubusercontent.com/63313732/154055032-c5d8992e-7feb-459a-8b3c-bed5807831f7.png">